tsconfig.json

```diff
@@ -15,10 +15,11 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "types": ["vite/client", "vite-plugin-pwa/client"]
+    "types": ["vite/client", "vite-plugin-pwa/client"],
+    "paths": {
+      "@careConfig": ["./care.config.ts"]
+    }
```

Comment:
```suggestion
    "paths": {
      "@careConfig": ["./care.config.ts"],
      "@/*": ["src/*"]
    }
```

Reply:
@rithviknishad @khavinshankar the usages will also have to be replaced. lets not do it in this PR